DependencyInjection/AbstractSmtpTransportConfigurator.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\SwiftmailerBundle\DependencyInjection;
+
+use Symfony\Component\Routing\RequestContext;
+
+/**
+ * Service configurator.
+ */
+class AbstractSmtpTransportConfigurator
```

Comment:
Because it configures subclasses of `Swift_Transport_AbstractSmtpTransport`. But I get your point, so I have stripped the `Abstract` prefix.